curriculum/challenges/english/25-front-end-development/quiz-css-layout-and-effects/66ed8ff4f45ce3ece4053eb4.md

```diff
@@ -325,131 +325,131 @@ Which CSS property is used to apply transformations such as rotation, scaling, a
+How is the total width of an element calculated in CSS when using the `border-box` value for the `box-sizing` property?
 
 #### --distractors--
 
-Total width = width + margin + padding + border
+Total Width = Content Width + Border + Margin
 
 ---
 
-Total width = width + margin + padding
+Total Width = Content Width + Padding + Border + Margin
 
 ---
 
-Total width = width + margin + border
+Total Width = Content Width
 
 #### --answer--
 
-Total width = width + left padding + right padding + left border + right border
+Total Width = Content Width + Padding + Border
 
 ### --question--
 
 #### --text--
 
-What does the filter property in CSS do?
+What is the purpose of the `filter` property in CSS?
 
 #### --distractors--
 
-It changes the layout of the element on the page
+Changing the scale and rotation of an element
 
 ---
 
-It changes the background color and texture of an element
+Changing the background color and texture of an element
 
 ---
 
-It alters the size and position of an element
+Altering the size and position of an element
 
 #### --answer--
 
-It applies visual effects like blur and saturation to an element
+Applying graphical effects, like blur and contrast, to an element
 
 ### --question--
 
 #### --text--
 
-How is the `filter` property specified in CSS?
+What does the CSS property `filter: grayscale(50%);` do to an element?
 
 #### --distractors--
 
-As only a single function, where percentage values must be used
+It increases the brightness of the element by 50%
 
 ---
 
-As `block` or `inline`, with values expressed in pixels
+It decreases the contrast of the element by 50%
 
 ---
 
-As `none` or multiple functions, but only percentage values are accepted
+It applies a sepia effect to the element
 
 #### --answer--
 
-As `none` or one or more functions, where percentage values can also be expressed as decimals
+It applies a 50% grayscale effect to the element
 
 ### --question--
 
 #### --text--
 
-What does the `contrast()` function in CSS do?
+What is the purpose of the `contrast()` function in CSS?
 
 #### --distractors--
 
-Adjusts the brightness of the image, where `0%` makes it black and `100%` makes it white
+Adjusting the brightness of the image, where `0%` makes it black and `100%` makes it white
 
 ---
 
-Changes the color balance of the image, where `0%` desaturates it and `100%` oversaturates it
+Adjusting the contrast of the image, where `100%` makes it grey, `0%` has no effect, and values over `100%` increase contrast
 
 ---
 
-Alters the opacity of the image, where `0%` makes it transparent and `100%` makes it opaque
+Altering the opacity of the image, where `0%` makes it transparent and `100%` makes it opaque
 
 #### --answer--
 
-Adjusts the contrast of the input image, where `0%` makes it grey, `100%` has no effect, and values over 100% increase contrast
+Adjusting the contrast of the image, where `0%` makes it grey, `100%` has no effect, and values over `100%` increase contrast
```

Comment:
It adjusts the contrast of the image, where `0%` makes it grey, `100%` has no effect, and values over `100%` increase contrast